src/nncf/quantization/algorithms/weight_compression/onnx_backend.py

```diff
@@ -428,16 +439,21 @@ def _replace_matmul_with_matmulnbits(
 
         original_matmul = self.name_to_node_map[weight_compression_parameters.node_with_weight.node_name]
 
-        activation_input_name = None
-        for input_name in original_matmul.input:
-            if input_name != weight_name:
-                activation_input_name = input_name
-        assert activation_input_name is not None, "Activation input name not found in original matmul node"
+        # Composing operation inputs: A, B, scales, zero_points[optional], g_idx[optional, deprecated], bias
+        bias_name = None
+        if weight_compression_parameters.node_with_weight.layer_attributes.has_bias():
+            bias_name = weight_compression_parameters.node_with_weight.layer_attributes.bias_attrs["name"]
```

Comment:
what if a name will be None and there is no bias - thus the None activation input will be filtered.. I think a filter list without none would be clearer
```suggestion
        non_activation_inputs = [weight_name]
        if weight_compression_parameters.node_with_weight.layer_attributes.has_bias():
            bias_name = weight_compression_parameters.node_with_weight.layer_attributes.bias_attrs["name"]
            non_activation_inputs.append(bias_name)
```